packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -46,6 +46,19 @@ This means that app will only be available for users that run Android SDK 20 or
                android:value="YOUR KEY HERE"/>
 ```
 
+#### Hybrid Composition
+
+To use [Hybrid Composition](https://flutter.dev/docs/development/platform-integration/platform-views) to render the `GoogleMap` widget on Android. Set
+`MethodChannelGoogleMapsFlutter.useAndroidViewSurface` to true.
```

Comment:
The use of method channels (and especially the fact that we are sharing the same impl between Android and iOS currently) is an implementation detail, so I would really like to not expose it. How about a trivial `AndroidGoogleMapsFlutter` subclass that can be exported and used here.